lib/Rex/PkgConf/Debian.pm

```diff
@@ -0,0 +1,105 @@
+#
+# (c) Jan Gehring <jan.gehring@gmail.com>
+#
+# vim: set ts=2 sw=2 tw=0:
+# vim: set expandtab:
+
+package Rex::PkgConf::Debian;
+
+use strict;
+use warnings;
+
+# VERSION
+
+use Rex::Helper::Run;
+
+use Rex::PkgConf::Base;
+use base qw(Rex::PkgConf::Base);
+
+sub new {
+  my $that  = shift;
+  my $proto = ref($that) || $that;
+  my $self  = $proto->SUPER::new(@_);
+
+  bless( $self, $proto );
+
+  return $self;
+}
+
+sub get_options {
+  my ( $self, $pkg, $option ) = @_;
+  die "Package name required to configure" unless $pkg;
+  my $conf_cmd = "debconf-show $pkg";
+
+  Rex::Logger::debug("Running command $conf_cmd");
+  my @lines = i_run $conf_cmd;
+
+  my %config;
+  for my $line (@lines) {
+    # Expecting: * postfix/relayhost: smtp.example.com
+    Rex::Logger::debug("Parsing line $line");
+    if ( $line =~ m!^(\*?)\h+(.+):\h*(.*)! ) {
+      my ($already_set, $question, $value) = ($1, $2, $3, $4);
```

Comment:
What is contained in `$4` at this point and/or why it gets omitted from the assignment?